packages/vats/src/core/coreEvalEnv.d.ts

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable */
+/**
+ * @file typesdef for the CoreEval environment
+ *
+ *   To use add this to the top of the proposal: /// <reference
+ *   types="@agoric/vats/src/core/coreEvalEnv"/>
+ *
+ *   That directive has to be before imports, but this one's only useful in
+ *   modules that have no imports or named exports.
+ */
+import type { VatData } from '@agoric/swingset-liveslots/src/vatDataTypes.js';
+import type { E, Far, getInterfaceOf, passStyleOf } from '@endo/far';
+
+// Provided by 'CORE_EVAL' handler in chain-behaviors.js
+declare global {
+  // @endo/far exports
+  var E: E;
+  var Far: Far;
+  var getInterfaceOfFar: getInterfaceOfFar;
+  var passStyleOfFar: passStyleOfFar;
+
+  // endowments
+  var VatData: VatData;
+  var assert: Assert;
+  var console: Console;
+}
```

Comment:
we're leaving `allPowers.modules` undocumented on purpose?

https://github.com/Agoric/agoric-sdk/blob/ba3b776444425bbcb275ce918794d9259f8e15da/packages/vats/src/core/boot-chain.js#L17-L20

_this comment got lost somehow_